src/main/java/frc/robot/intake/IntakeSubsystem.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.intake;
+
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.intake.commands.IntakeCommand;
+
+public class IntakeSubsystem extends SubsystemBase {
+  private final CANSparkMax motor;
+  private IntakeMode mode = IntakeMode.STOPPED;
+
+  /** Creates a new IntakeSubsystem. */
+  public IntakeSubsystem(CANSparkMax motor) {
+    this.motor = motor;
+
+    this.setDefaultCommand(
```

Comment:
@jonahsnider I thought we agreed to having the default command be in RobotContainer, for clarity purposes. Is there a reason we want to switch this paradigm?

Reply:
good catch, i forgot we had decided on default commands in robotcontainer. and this code was written before we talked about that i believe.